### ISSUE TITLE
Spamming error message when no camera is detected

### ISSUE THREAD

**Issue by JuliaSzulc** [suhren/camscan#2]
When the program is launched without any camera connected it endlessly fills the log with `ERROR:root:Can't receive frame (stream end?). Exiting ...`. Would be good to remove the loop.

**Comment by suhren** [comment]
Fair point. It was useful when debugging in the beginning but not really needed now. The logging statement is removed in this commit: https://github.com/suhren/camscan/commit/f70973d4f11c48f6068015bed9cff1b2eae8bfe0
